src/components/layout/SignUp.jsx

```diff
@@ -18,21 +52,63 @@ const SignUp = ({ switchForm }) => {
       </section>
 
       <div className={styles.forms}>
```

Comment:
I usually use a form instead of div for something like this because I believe it is best practice.

Reply:
John worked on all the design aspects. I just added the API and made some changes accordingly. I’ll check with John about this and make any necessary updates. Thank you